src/hooks/ens.ts

```diff
@@ -0,0 +1,28 @@
+import { SafeAppProvider } from "@gnosis.pm/safe-apps-provider";
+import { useSafeAppsSDK } from "@gnosis.pm/safe-apps-react-sdk";
+import { ethers } from "ethers";
+import { useMemo } from "react";
+
+export interface EnsResolver {
+  /**
+   * Resolves a ENS name to a corresponding address.
+   * Important: If the name is already a valid address, this address will be returned.
+   *
+   * Returns null if the ENS name cannot be resolved.
+   *
+   * @param ensName ENS Name or address.
+   */
+  resolveName(ensName: string): Promise<string | null>;
```

Comment:
I was kind of expecting this to take an address and return a name. I'm not sure I understand where we are getting the name from in the first place.